app/res/values-v11/themes.xml

```diff
@@ -81,6 +81,8 @@
         <item name="android:paddingLeft">@dimen/title_paddingLeft</item>
 	</style>
     
-    
+    <style name="Theme.Dialog" parent="@android:style/Theme.Holo.Light.Dialog">

```

Comment:
I can't remember exactly, but I think it was to improve the styling of a file selection dialog that would pop up during the process. I'm sure the flow has changed considerably since I implemented it, so if everything looks fine, it should be OK to remove.